src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -51,6 +51,8 @@ public class EditCommand extends Command {
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_INVALID_TAG = "One or more tag(s) is not in Tag List. "
```

Comment:
Would it be better to display which tag is not present in the tagList, instead of indicating one or more tags?

Reply:
changed as per comment